reporters_db/data/reporters.json

```diff
@@ -16167,6 +16153,7 @@
             ],
             "name": "Michigan Reports",
             "variations": {
+                "M.,": "Mich.",
```

Comment:
I dont think the comma should be included

Reply:
You are right, i copy/paste the variation, regex already includes the comma

Reply:
great can you update this